assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js

```diff
@@ -61,33 +61,7 @@ function PaginatedMediaGallery({
   setNextPage,
 }) {
   // TODO(#1698): Ensure scrollbars auto-disappear in MacOS.
```

Comment:
We can remove this TODO as well (That issue has also been closed)

Reply:
Done.